src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DataStrictnessValidation.feature

```diff
@@ -0,0 +1,355 @@
+Feature: Data strictness
+    Validate that the API meets data strictness / laxity requirements.
```

Comment:
I've done a lot of refactoring in this file compared to the original commit. If you look at the commits one by one then you will see the refactoring steps in detail.